src/main/resources/messages/MessageBundle_ru.properties

```diff
@@ -70,4 +65,7 @@ settings.general.content.sticker.ignore.scaling=Игнорировать мас
 settings.general.randomizer.install=Выбирать тему случайно
 settings.general.sticker.hide-on-hover=Скрыть при наведении
 settings.general.sticker.hide.delay=Скрыть при задержке (ms)
-doki.settings.allow.promotion=Allow Promotional Content
\ No newline at end of file
+doki.settings.allow.promotion=Allow Promotional Content
```

Comment:
@Dragon-0609 I've added some more of these, if you could be a pal and translate them, that be great. Thanks!